docs/website/docs/building-from-source/ios.md

```diff
@@ -0,0 +1,130 @@
+# Cross-compilation for iOS
```

Comment:
That doc should look similar in structure to the Android doc, so I'd change this title to "iOS cross-compilation"

```suggestion
# iOS cross-compilation
```